driver/tests/res/translation/with_src_files/reference/mod11.xast

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is `.xast` extension introduced with the new driver?

Reply:
".xast" is Fortran fully transformed to XCodeML. Introduced to distinguish from ".xmod". However, it is internal to CLAW.